beets/util/__init__.py

```diff
@@ -776,16 +776,20 @@ def interactive_open(targets, command):
 
     Can raise `OSError`.
     """
-    # Split the command string into its arguments.
-    try:
-        command = shlex_split(command)
-    except ValueError:  # Malformed shell tokens.
-        command = [command]
-    command.insert(0, command[0])  # for argv[0]
+    if command:
+        # Split the command string into its arguments.
+        try:
+            args = shlex_split(command)
+        except ValueError:  # Malformed shell tokens.
+            args = [command]
+    else:
+        args = [open_anything()]
```

Comment:
That won't work because the `command` parameter is being *passed*, it is just being explicitly set to a falsey value. So using a default argument won't work.